cpp/ql/src/semmle/code/cpp/ir/dataflow/internal/DataFlowUtil.qll

```diff
@@ -166,6 +170,16 @@ private class ThisParameterNode extends InstructionNode {
   override string toString() { result = "this" }
 }
 
+class VarArgParameterNode extends ParameterNode {
+  override VarArgsStartInstruction instr;
+
+  override predicate isParameterOf(Function f, int i) {
+    i instanceof ParameterIndex and
```

Comment:
Can we restrict `i` even more? Like restricting it to the max argument number passed to this particular function? (can virtually-dispatched functions use varargs?)

Reply:
Looks like it: https://ideone.com/n2qdFZ

Reply:
Okay. Then I'm fine with keeping `i instanceof ParameterIndex` for now.